reviewrot.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+import yaml
+import logging
+import logging.config
+from reviewrot import get_git_service
+from os.path import expanduser
+
+def main(fltr, value, duration):
+    """
+    starts execution
+    """
+    #TODO: Implement StreamHandler
+    # logging.config.fileConfig('log.conf')
+    log = logging.getLogger(__name__)
+    home = expanduser('~')
+    # read input from home directory for pull requests
+    with open(home + '/.config.yaml', 'r') as f:
+            config = yaml.load(f)
+    
+    #print config
+    for item in config['git']:
+        git = item.keys()[0].lower()
```

Comment:
See my comments in the config file. The extra `0` index used here and in a few other spots isn't necessary.